.github/workflows/main.yml

```diff
@@ -34,8 +42,10 @@ jobs:
     stats:
         runs-on: ubuntu-latest
         steps:
-            - uses: actions/setup-node@v1
             - uses: actions/checkout@v2
+            - uses: actions/setup-node@v2
+              with:
+                node-version: '10'
```

Comment:
@julien just to confirm after we merge it we can send a new PR changing this to use the LTS version, right?